js/preview/src/build.ts

```diff
@@ -43,23 +43,25 @@ export async function make_build({
 							emptyOutDir: true,
 							outDir: join(template_dir, entry),
 							lib: {
-								entry: join(source_dir, entry, "index.ts"),
+								entry: join(source_dir, path),
```

Comment:
typo didn't matter however because we were look for `Index.svelte/index.ts` and `Example.svelte/index.ts` which is nonsensical.